posix-configs/SITL/init/ekf2/hippocampus

```diff
@@ -0,0 +1,36 @@
+uorb start
+param load
+param set MAV_TYPE 3
+param set SYS_AUTOSTART 4010
+param set SYS_RESTART_TYPE 2
+param set COM_DISARM_LAND 0
+dataman start
+param set CAL_GYRO0_ID 2293768
+param set CAL_ACC0_ID 1376264
+param set CAL_ACC1_ID 1310728
+param set CAL_MAG0_ID 196616
+
+simulator start -s
+
+pwm_out_sim mode_pwm
+mixer load /dev/pwm_output0 ROMFS/px4fmu_test/mixers/uuv_quad_x.mix
+
+gyrosim start
+accelsim start
+barosim start
+adcsim start
+
+sleep 1
+
+sensors start
+commander start
+
+attitude_estimator_q start
```

Comment:
so where do you start the `ekf2`? and if this is to start the ek2, why is there the `attitude_estimator_q`? Seems a mistake to me.

Reply:
Take a look how the other ekf2 config files are written

Reply:
`ekf2 start`

Also you are not starting any of the attitude/position controllers. Are you able to control the vehicle at all?

Reply:
Hi, I can change the attitude_estimator_q to the ekf2 for the start-up script in the ekf2 folder. I have already tested it and it worked. Still I am not starting any of the attitude/position controller since I have written one in order to follow given trajectories or hold the position and orientation (based on the attitude controller from PX4). Also I have written an trajectory planner which, hopefully, should be able at the end of september to generate local optimal trajectories along waypoints by using an adaption of the STOMP algorithm. Another problem I haven't been able to solve yet is that I do not receive position data from the ekf2, thus the x,y,z coordinates in the control states are all near zero all the time. In order to get position data I send the generated position data from gazebo with ROS over mavros / mavlink to the Firmware (Therefore I use the att_pos_mocap topic). This is not the best solution, thus if you have an idea how I can get position data from gazebo to the ekf2 please let me know.
Best regards
Nils